### ISSUE TITLE
PHP 8.3.4 with zend_extension=opcache.so will get error

### ISSUE THREAD

**Issue by hetao29** [neo4j-php/neo4j-php-client#203]
**Describe the bug**
Laudis\Neo4j\Databags\SummaryCounters::__construct(): Argument #14 ($systemUpdates) must be of type int, false given, called in vendor/laudis/neo4j-php-client/src/Formatter/SummarizedResultFormatter.php on line 151

**System inffo**
 - PHP version: 8.3.4
 - laudis/neo4j-php-client: 3.0.6
 - neo4j: v4 or v5
 - Ubuntu 22.04.4 LTS

** resolve 2 ways:
 - 1. comment the php.ini #zend_extension=opcache.so or remove the line;
 - 2. change the file vendor/laudis/neo4j-php-client/src/Formatter/SummarizedResultFormatter.php line 150
 ```php
return new SummaryCounters(
            $stats['nodes-created'] ?? 0,
            $stats['nodes-deleted'] ?? 0,
            $stats['relationships-created'] ?? 0,
            $stats['relationships-deleted'] ?? 0,
            $stats['properties-set'] ?? 0,
            $stats['labels-added'] ?? 0,
            $stats['labels-removed'] ?? 0,
            $stats['indexes-added'] ?? 0,
            $stats['indexes-removed'] ?? 0,
            $stats['constraints-added'] ?? 0,
            $stats['constraints-removed'] ?? 0,
            $updateCount > 0,
            ($stats['contains-system-updates'] ?? $stats['system-updates'] ?? 0) >= 1,
            systemUpdates:$stats['system-updates'] ?? 0
        );
```

**Comment by hetao29** [comment]
I change the code below
```php
$results = $client->runStatements($statements);
```
to
```php
$tsx = $client->beginTransaction($statements);
$tsx->commit();
```
And It's OK.

**Comment by zorac** [comment]
I've seen the same error, but only very occasionally:
`TypeError: Laudis\Neo4j\Databags\SummaryCounters::__construct(): Argument #14 ($systemUpdates) must be of type int, false given, called in /app/vendor/laudis/neo4j-php-client/src/Formatter/SummarizedResultFormatter.php on line 150`

System info:
* PHP version: 8.3.21 (cli)
* laudis/neo4j-php-client: 3.2.0
* neo4j: 5.25.1 (enterprise cluster)
* Alpine Linux 3.21.3 (in a Docker container)

The first time I saw this was after upgrading to neo4j-php-client 3.3.0 shortly after its release in a test environment, and downgrading resolved it. I suspect now that the version change was a coincidence.

The second time I've observed it was today, and the errors were coming from only one of 40 containers running the same image, and look to have begun shortly after it was started. Stopping that container and allowing it to be recreated cleared the issue. I've not seen it previously in several months of running that production environment, so I don't see any obvious path to being able to reproduce it.

For reference, the calling code is:
```php
        return $this->getClient()->writeTransaction(
            fn ($tsx) => $tsx->run($query, $parameters)
        );
```
